src/V3Timing.cpp

```diff
@@ -345,19 +388,33 @@ class TimingSuspendableVisitor final : public VNVisitor {
             if (depVxp->nodep()->user2() & T_SUSPENDEE) propagateFlags(depVxp, T_SUSPENDEE);
         }
         if (dumpGraphLevel() >= 6) m_suspGraph.dumpDotFilePrefixed("timing_deps");
-        // Propagate process
+
+        // Propagate needProcess
```

Comment:
```suggestion
        // Propagate NEED_PROC
```
Let's keep the names consistent.

Reply:
Done.